CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.28.0 / 2023-04-17
+
+- [FEATURE] Revamp project page (#1062)
+- [ENHANCEMENT] Highlight behavior slight improvement using opacity blur (#1107)
+- [ENHANCEMENT] Increase tooltip max height to fit more series w/o scrolling (#1111)
+- [BREAKINGCHANGE] Define queries in panel spec instead of panel plugin spec (#1032)
```

Comment:
I'm not sure if an outside observer would know exactly how to handle this breaking change. I'm not sure if some extra notes should go in here or in the description on the linked PR. Leaning towards the latter, but I'm curious what other folks think.

Reply:
I think adding more details in the linked PR should be ok?